### DISCUSSION TITLE
Pc frozens when tries to sync

### DISCUSSION THREAD

**Discussion by mariorangel777** [Chia-Network/chia-blockchain#3520] [Support]
Since the last Friday, I have a lot of issues with syncing, Now I have the problem that the whole pc froze when chia tries to get synced, always stops at 50xxx/224863. I opened port 8444 in the firewall and the modem. I'm feeling really frustrated cause I recently started and now I can't even try to farm or plotting.

**Comment by mariorangel777** [comment]
and another thing, the full node connections are 4, I try to erase all not one, but always have all these 4 connections to full node

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
